templates/default/partials/version/download.html.twig

```diff
@@ -53,9 +59,19 @@ ddev launch</code></pre>
                                     <pre><code>winpty ddev typo3cms install:setup</code></pre>
```

Comment:
I guess this line needs adjustments as well.

Reply:
Thanks, has now been taken into account